src/main/java/com/innohub/innosorter/authentication/AuthenticationService.java

```
@@ -0,0 +1,41 @@
+package com.innohub.innosorter.authentication;
+
+public class AuthenticationService {
+
+	protected String nullPasswordMessage = "Null Password";
+	RegistrationServiceDao registrationServiceDao = new RegistrationServiceDao();
+	protected static String unsuccessfulPasswordUpdateMessage = "Same as old password";
+	protected static String successfulPasswordUpdateMessage = "Password has been updated";
+	protected static String invalidUserMessage = "Invalid Username";
+	protected static String invalidPasswordMessage = "Invalid Password";
+	
+	public String addNewUser(String username, String password) {
+		
+		//if the password is empty, return false
+		if(password.isEmpty()){
```

Would you please throw Exception instead of returning false